IssuesData.RepositoryUrl is missing

IssuesData.RepositoryUrl is missing:

```
C:/Git/Azure/Cake.Issues.Recipe-Demo/tools/Cake.Issues.Recipe.0.2.1/Content/tasks/pullrequestsystems/AzureDevOpsPullRequestSystem.cake(28,22): error CS1061: 'IssuesData' does not contain a definition for 'RepositoryUrl' and no accessible extension method 'RepositoryUrl' accepting a first argument of type 'IssuesData' could be found (are you missing a using directive or an assembly reference?)
```

--- a/Cake.Issues.Recipe/Content/data/IssuesData.cake
+++ b/Cake.Issues.Recipe/Content/data/IssuesData.cake
@@ -55,7 +55,10 @@ public class IssuesData
         this.RepositoryRootDirectory = context.MakeAbsolute(context.Directory("./"));
 
         this.BuildServer = DetermineBuildServer(context);
-        this.PullRequestSystem = DeterminePullRequestSystem(context, BuildServer.DetermineRepositoryRemoteUrl());
+        this.PullRequestSystem =
+            DeterminePullRequestSystem(
+                context,
+                BuildServer.DetermineRepositoryRemoteUrl(context, this.RepositoryRootDirectory));
     }
 
     /// <summary>

--- a/Cake.Issues.Recipe/Content/tasks/pullrequestsystems/AzureDevOpsPullRequestSystem.cake
+++ b/Cake.Issues.Recipe/Content/tasks/pullrequestsystems/AzureDevOpsPullRequestSystem.cake
@@ -25,7 +25,7 @@ public class AzureDevOpsPullRequestSystem : BasePullRequestSystem
         context.ReportIssuesToPullRequest(
             data.Issues,
             context.TfsPullRequests(
-                data.RepositoryUrl,
+                data.BuildServer.DetermineRepositoryRemoteUrl(context, data.RepositoryRootDirectory),
                 data.PullRequestId.Value,
                 context.TfsAuthenticationOAuth(context.EnvironmentVariable("SYSTEM_ACCESSTOKEN"))),
             data.RepositoryRootDirectory);
@@ -52,7 +52,7 @@ public class AzureDevOpsPullRequestSystem : BasePullRequestSystem
 
         var pullRequestSettings =
             new TfsPullRequestSettings(
-                data.RepositoryUrl,
+                data.BuildServer.DetermineRepositoryRemoteUrl(context, data.RepositoryRootDirectory),
                 data.PullRequestId.Value,
                 context.TfsAuthenticationOAuth(context.EnvironmentVariable("SYSTEM_ACCESSTOKEN")));
 

